continuous-testing/starter-kits/README.md

```diff
@@ -1,3 +1,55 @@
 # Starter Kits
 
-Nothing here yet, come back soon... 
\ No newline at end of file
+This page contains starter kit information, which represent templates, code and configuration to help you get started quickly with continuous testing best practices described in this overall guide. Please see categories and links below for details. 
+
+## Software Composition Analysis
+
+This section contains links to sample actions, templates and configurations that analyze and validate composition of Open Source Software (OSS) components in software systems. Identifying software and licensing vulnerabilites and ensuring routine software updates is an [OSS cybersecurity](https://www.cisa.gov/uscert/ncas/alerts/aa22-137a) [best practice](https://appel.nasa.gov/2022/06/30/spotlight-on-lessons-learned-open-source-and-commercial-web-software-vulnerabilities/).  
+
+### Dependabot
```

Comment:
I have a suggested adjustment to this writeup. I ran through the steps and the following came to mind:
- Having two sections for Dependabot (dependency updates vs security) is a bit confusing. Running through the guide using the GitHub Settings UI, I saw the toggles for Dependabot options like "Dependabot security updates", "Dependabot version updates", and "Dependabot alerts" all together, and I was confused why this guide was only talking about only two of the features and doing so separately in different sections. 
- For the "Dependabot version updates" - the Admin UI approach still requires creation of a manual `.github/dependabot.yml` file - so I don't know if its accurate to say there are two approaches, its really only one: create the `dependabot.yml` file and then go to the Admin UI and click "Enable" for "Dependabot version updates".
- It might help to keep it simple and focus on the essentials here: (1) knowledge that Dependabot exists and references to its documentation on installing, and (2) sample `dependabot.yml` config files with opinionated settings people can use straight away for their language-specific project

See adjusted write-up below (feel free to modify and let me know what you think):

### Dependabot
A GitHub ecosystem tool for dependency version and security vulnerability analysis. 

Before you install / configure [Dependabot](https://docs.github.com/en/code-security/dependabot), discuss with your development team the cybersecurity best practice to regularly update OSS to latest versions and seek consensus on a workflow to accept proposed updates.

#### Requirements

Admin rights are necessary to modify `Code security and analysis` settings.

#### Installation

1. Navigate to your GitHub repository, then navigate to [`Settings` **->** `Code security and analysis` **->** `Dependabot`]
2. We recommend enabling the following settings (see screenshot):
   a. "Dependabot alerts"
   b. "Dependabot security updates"
   c. "Dependabot version updates" - requires further configuration to Enable, see step 3.
<img width="667" alt="Screen Shot 2023-01-09 at 4 11 08 PM" src="https://user-images.githubusercontent.com/3129134/211433166-a9d0adc2-96d7-4c8b-af5b-e624cde2aad9.png">

3. For "Dependabot version updates", you'll need to ensure you've committed a `.github/dependabot.yml` file. See our examples:
   a. [Python example](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/dependabot.yml)
   b. Java example (coming soon)
4. Dependabot is installed. You can monitor via:
   a. Dependency updates: [`Insights` **->** `Dependency graph`]
   b. Security updates: [`Security` **->** `Dependabot`]
   c. Automated pull requests: [`Pull requests` **->** `Author: `is:open is:pr author:app/dependabot `]

Reply:
Change in approach applied.